package.json

```diff
@@ -8,8 +8,9 @@
     "scripts/github-bot"
   ],
   "scripts": {
-    "build": "yarn clean && npm-run-all build:ts build:sass",
+    "build": "yarn clean && npm-run-all build:ts build:vscode-extension-dependencies build:sass",
```

Comment:
is this necessary?

Reply:
please see latest commit. I have excluded build:vscode-extension-dependencies from main build because this scrip os not required for local development. 
I have moved build:vscode-extension-dependencies to .github/workers. Not sure if I missed any worker. Please see